Directory.Build.props

```diff
@@ -80,19 +88,17 @@
   </PropertyGroup>
 
   <!-- Default properties for CI builds -->
-  <!-- TODO: Remove obsolete condition when moved to arcade. -->
-  <PropertyGroup Condition="'$(IsCIBuild)' == 'true' OR '$(ContinuousIntegrationBuild)' == 'true'">
+  <PropertyGroup Condition="'$(IsTestProject)' == 'true' and '$(ContinuousIntegrationBuild)' == 'true'">
     <WithoutCategories>IgnoreForCI</WithoutCategories>
     <EnableDumpling>true</EnableDumpling>
     <CrashDumpFolder Condition="'$(RunningOnUnix)' != 'true'">%TMP%\CoreRunCrashDumps</CrashDumpFolder>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(ContinuousIntegrationBuild)' == 'true'">
```

Comment:
As discussed in another PR, does `ContinuousIntegrationBuild` also include officials builds?

Reply:
Not yet but it likely may end up being set for official builds as well.

Reply:
For these do you want them to be part of the official builds as well?

Reply:
yes if you could condition it for official builds as well, that would be great.